### ISSUE TITLE
Black Guard Ability [Crimson Death] Not Working as Intended

### ISSUE THREAD

**Issue by Haojun-Phalanx** [WarEmu/WarBugs#19243]
![crimsondeath2](https://user-images.githubusercontent.com/23129933/123400108-121f5080-d5ae-11eb-9aeb-d32a58bef673.JPG)

Crimson Death's debuff part is undefendable, there is no info about it on tooltip.

![crimsondeath](https://user-images.githubusercontent.com/23129933/123400213-2f541f00-d5ae-11eb-9550-95d6f0bc63d6.jpg)

**Comment by GK111** [comment]
https://www.returnofreckoning.com/forum/viewtopic.php?f=42&t=32595

>Blackguard :
>- Crimson Death damage component will now correctly Check for defense but the debuff remain undefendable like on live

**Comment by Haojun-Phalanx** [comment]
> https://www.returnofreckoning.com/forum/viewtopic.php?f=42&t=32595
> 
> > Blackguard :
> > 
> > * Crimson Death damage component will now correctly Check for defense but the debuff remain undefendable like on live

yes, i'm aware of that patchnote hence i posted the "live" tooltip.

**Comment by LiuBu1** [comment]
https://www.returnofreckoning.com/forum/viewtopic.php?f=42&t=32595&start=30
Grunbag- Only the debuff is undefendable , damage component check for defense (like on live, according to mythic setting in the client).

**Comment by Haojun-Phalanx** [comment]
again i don't know if it's intended or not, it's just not **logical** to get debuffed if you "defend" the ability. there are only 3 abilities acting like this atm, you can check my other bugreports.
